pkg/email/config.go

```diff
@@ -0,0 +1,16 @@
+package email
+
+var hostName = "mint.cms.gov"
```

Comment:
Could the contents of this file be inside `pkg/shared/oddmail/email_service_config.go`? Seems like there's 2 spots in the codebase that hold email related config with this file and that one.

Reply:
I've been thinking about this over the weekend. Definitely agree. Wondering if it's best to load the configs then overwrite some with environment vars before injecting into the email service. Will ping you about this in Slack to get your thoughts.